services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchRulesSpec.groovy

```diff
@@ -82,9 +82,14 @@ class SwitchRulesSpec extends BaseSpecification {
         requireProfiles("virtual")
 
         given: "A switch with some rules installed (including default) and not connected to the controller"
+        def srcSwitchDefaultMeters = northbound.getAllMeters(srcSwitch.dpId)
         def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
         flowHelper.addFlow(flow)
 
+        def srcSwitchCreatedMetersIds = northbound.getAllMeters(srcSwitch.dpId).meterEntries.findAll {
+            !srcSwitchDefaultMeters.meterEntries*.meterId.contains(it.meterId)
```

Comment:
No need to obtain `srcSwitchDefaultMeters` on L85. Just do 
```
def srcSwitchCreatedMetersIds = northbound.getAllMeters(srcSwitch.dpId).meterEntries.*meterId
   .findAll { it >= MIN_FLOW_METER_ID } 
```

Even no need to do it here, you can call for this directly during cleanup procedure